app/models.py

```diff
@@ -1848,7 +1852,12 @@ def serialize(self):
     SEND_LETTERS,
     MANAGE_API_KEYS,
     PLATFORM_ADMIN,
-    VIEW_ACTIVITY]
+    VIEW_ACTIVITY,
+    CREATE_BROADCASTS,
```

Comment:
The button to create a broadcast is labelled `Prepare broadcast` should we use `prepare_broadcast`. I don't have a strong feeling but am wondering if reflecting the wording on the UI is better.

Reply:
We’re probably going to review the language in the UI so I think `create` is OK.